src/pages/admin/ApplicationManagement/ManageCategoryItem.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { CategoryItemWrapper, TitleContainer, ItmeName, ItemCount, RentalInfoWrapper, RentalInfoContent, RentalInfo } from './ManageCategoryItem.styles';
+
+const ManageCategoryItem = ({ itemData }) => {
+  return (
+    <CategoryItemWrapper>
+      <TitleContainer>
+        {/* 블루투스 키보드&마우스 세트 아이콘 만들어지기 전까지 임시 이미지*/}
+        <img src={itemData.imgSrc} alt="블루투스 키보드&마우스 세트 아이콘" />
+        <ItmeName>{itemData.itemName}</ItmeName>
```

Comment:
```suggestion
        <ItemName>{itemData.itemName}</ItemName>
```

ide가 보여주는 경고를 잘 보셔유~